polar-core/src/vm.rs

```diff
@@ -825,6 +882,16 @@ impl PolarVirtualMachine {
             "Called isa with bare dictionary!"
         );
 
+        if self.polar_log {
+            let relevant_bindings = self.relevant_bindings(&[left, right]);
+            self.log(&format!(
+                "ISA:'{}' matches '{}', BINDINGS: {:?}",
+                left.to_string(),
+                right.to_string(),
+                relevant_bindings
```

Comment:
Could we do something more like:
```suggestion
                "ISA: {} matches {}, BINDINGS: {}",
                left.to_polar(),
                right.to_polar(),
                format_bindings(relevant_bindings)
```
I find the single quotes around Polar terms distracting, and the bindings should be non-debug, too, right?

Reply:
:+1: